e2e/components/Accordion/Accordion-test.avt.e2e.js

```diff
@@ -144,4 +157,36 @@ test.describe('@avt Accordion', () => {
       'Accordion @avt-advanced-states disabled'
     );
   });
+
+  test('@avt-keyboard-nav expand/collapseAll', async ({ page }) => {
+    await visitStory(page, {
+      component: 'Accordion',
+      id: 'components-accordion--controlled',
+      globals: {
+        theme: 'white',
+      },
+    });
+    const expand_all_btn = page.getByRole('button', {
+      name: 'Click to expand all',
+    });
+    const collapse_all_btn = page.getByRole('button', {
+      name: 'Click to collapse all',
+    });
+    const accordion_btn1 = page.getByRole('button', {
+      name: 'Section 1 title',
+    });
+    await expect(expand_all_btn).toBeVisible();
+    await expect(collapse_all_btn).toBeVisible();
+    await page.keyboard.press('Tab');
+
+    // Check the focus on Expand button
+    await expect(expand_all_btn).toBeFocused();
+    await page.keyboard.press('Enter');
+    await expect(accordion_btn1).toHaveAttribute('aria-expanded', 'true');
+    //focus on collapse button
+    await page.keyboard.press('Tab');
+    await expect(collapse_all_btn).toBeFocused();
+    await page.keyboard.press('Enter');
+    await expect(accordion_btn1).toHaveAttribute('aria-expanded', 'false');
+  });
```

Comment:
Honestly I think we could remove this from the e2e tests here since you've got a nearly identical test within the jest unit tests in `packages/react/src/components/Accordion/__tests__/Accordion-test.js`

```suggestion
```

Reply:
Changes Done as per suggestions.